config/ons.yml

```diff
@@ -23,4 +23,4 @@ entries:
   replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/Older_deprecated/ons_release_v1.1.obo
   
 - prefix: /imports/
-  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/imports/ONS_imports.owl
+  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/imports/
```

Comment:
This should not point to "https://raw.githubusercontent.com" as now this is a folder

Reply:
Thats ok: this is not an exact purl to some file! It is a prefix pattern, so you wouldn't just try to go to /imports/, you would always look for a specific file like:

http://purl.obolibrary.org/obo/ons/imports/ONS_imports.owl

The purl rule here would make sure that if you look for that (the above example), it would redirect to raw github.

Reply:
Ok thanks!